ansible/roles/telemetry-installer/scenarios/install_telemetry_tools.yml

```diff
@@ -13,13 +13,21 @@
 # limitations under the License.
 
 ---
+# ----Stop all services to all ansible re-run in case of failure-----
+- name: stop all services
+  shell: "{{ item }}"
+  with_items:
+    - systemctl stop prometheus node_exporter alertmanager pushgateway sysstat lvm_exporter
+  become: yes
+  ignore_errors: yes
+
 # ---------start to install prometheus---------
 - name: add users and give permissions for prometheus
   shell: "{{ item }}"
   with_items:
-    - useradd --no-create-home --shell /bin/false prometheus
-    - mkdir /etc/prometheus
-    - mkdir /var/lib/prometheus
+    - id -u prometheus || useradd --no-create-home --shell /bin/false prometheus 
```

Comment:
Please move the blank space in the end of this line.

Reply:
Removed extra space